sdk/src/schema.ts

```diff
@@ -88,7 +88,7 @@ export type RemovePositionTxResult = {
 
 export type SwapTxResult = {
   hash: string
-  events: [CrossTickEvent, SwapEvent]
+  events: (CrossTickEvent | SwapEvent)[]
```

Comment:
Are you sure you want to use an array here? I recently described the difference here: https://github.com/invariant-labs/protocol-a0/pull/188#discussion_r1467271880

Perhaps consider using a union pattern like this:
Maybe pattern using union?
```Typescript
events: [CrossTickEvent, SwapEvent] | [SwapEvent]
```